frontend/package.json

```diff
@@ -17,14 +17,21 @@
     "react-dom": "^17.0.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.9.6",
-    "@babel/preset-env": "^7.9.6",
-    "@babel/preset-react": "^7.9.4",
-    "babel-loader": "^8.1.0",
```

Comment:
As with CCM, I'm just relying on `tsc` (the TypeScript compiler) to do the work that we used to use Babel to do. See https://www.typescriptlang.org/docs/handbook/babel-with-typescript.html for some additional info.